src/main/connection.cpp

```diff
@@ -212,17 +212,27 @@ std::unique_ptr<PreparedStatement> Connection::prepareNoLock(
 std::string Connection::getNodeTableNames() {
     lock_t lck{mtx};
     std::string result = "Node tables: \n";
+    std::vector<std::string> nodeTableNames;
     for (auto& tableIDSchema : database->catalog->getReadOnlyVersion()->getNodeTableSchemas()) {
-        result += "\t" + tableIDSchema.second->tableName + "\n";
+        nodeTableNames.push_back(tableIDSchema.second->tableName);
+    }
+    std::sort(nodeTableNames.begin(), nodeTableNames.end());
```

Comment:
do we need to sort. We should print in the same order as defined in catalog

Reply:
@acquamarin?

Reply:
We don't preserve the order of properties inside the catalog, we used unordered_map to store the nodeTableSchemas